sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 # Release History
 
-## 4.0.0-beta.3 (Unreleased)
+## 4.0.0-beta.3 (2020-11-12)
 
+### New Features
+- Added support for passing a `ServiceVersion` in clients and their builders.
+
+### Changes
```

Comment:
Let's make this one of these https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change
or was it intentionally kept so ?